plugins: Add Youtube-HLS to the Service dropdown

mv2731 opened the pull request:
This commit changes the name of the option called Youtube
/ Youtube Gaming to Youtube-RTMP to be more specific
and adds a Youtube-HLS option in the Stream settings
Service dropdown.

mv2731 committed:
plugins: Add Youtube-HLS to the Service dropdown

This commit changes the name of the option called Youtube
/ Youtube Gaming to Youtube-RTMP to be more specific
and adds a Youtube-HLS option in the Stream settings
Service dropdown.

ushadow commented on plugins/rtmp-services/data/package.json:
```diff
@@ -1,10 +1,5 @@
-			"version": 132
-		}
-	]
+	"url" : "https://obsproject.com/obs2_update/rtmp-services",
```
Is the formatting change produced by clang-format?

mv2731 replied:
I ran clang-format -i package.json on the file, so probably? but I can check...also when I run this I see
![image](https://user-images.githubusercontent.com/60421398/83559318-63136100-a4e2-11ea-8e6c-f1d4d9c4f82e.png)

mv2731 committed:
Revert "plugins: Add Youtube-HLS to the Service dropdown"

This reverts commit 92e35faf89154d83c772ca3e68d47a9561c2f1f7.

mv2731 committed:
plugins: Add Youtube-HLS to the Service dropdown

This commit changes the name of the option called Youtube
/ Youtube Gaming to Youtube-RTMP to be more specific
and adds a Youtube-HLS option in the Stream settings
Service dropdown.

ushadow commented on plugins/rtmp-services/data/services.json:
```diff
@@ -198,7 +198,27 @@
+                },
+                {
+                    "name": "Backup YouTube ingest server",
+                    "url": "https://b.upload.youtube.com/http_upload_hls?cid=<stream key>&copy=0&file=out.m3u8"
```
copy=1

ushadow commented on plugins/rtmp-services/data/services.json:
```diff
@@ -198,7 +198,27 @@
+            "servers": [
+                {
+                    "name": "Primary YouTube ingest server",
+                    "url": "https://a.upload.youtube.com/http_upload_hls?cid=<stream key>&copy=0&file=out.m3u8"
```
cid={stream_key}

And same below.

mv2731 committed:
plugins: Fix URLs of Youtube-HLS servers
